internal/own/codeowners/proto/match_test.go

```diff
@@ -0,0 +1,135 @@
+package proto_test
+
+import (
+	"reflect"
+	"testing"
+
+	codeownerspb "github.com/sourcegraph/sourcegraph/internal/own/codeowners/proto"
+)
+
+type testCase struct {
+	pattern string
+	paths   []string
+}
+
+func TestMatch(t *testing.T) {
```

Comment:
Maybe rename the function to reflect the function being tested? Like this or something similar?
```suggestion
func TestFileOwners_Match(t *testing.T) {
```

Reply:
Good idea. I'll do `TestFileOwnersMatch` without the underscore though. I recently got a suggestion (I think from @sashaostrikov) about the elegance of keeping the casing more consistent.